### DISCUSSION TITLE
[Hackathon Use Case] - ESMA templates transformations for NON-EU originators/providers

### DISCUSSION THREAD

**Discussion by maoo** [finos/finos-pmcs#246] [Ideas]
# ESMA templates transformations for NON-EU originators/providers
_From @CalogeroZarbo @lucaborella89 (algoritmica.ai)_

## Business Problem
In the context of the new EU Securitisation Regulation, NON-EU originators and / or NON-EU specialist reporting providers now have to report loan-level data using ESMA templates if they want to sell securities to EU investors. See [here](https://www.srz.com/resources/esma-template-reporting-to-be-required-for-eu-institutions-which.html) more info. Some US GSE (agency) transactions are already being mapped to ESMA templates (see [here](https://capitalmarkets.fanniemae.com/media/7861/display)).

This is going to be even more relevant for non-agency transactions, which rely more on the EU investors base. The problem is that non-agency transactions come with custom loan-level data templates (i.e. every bank has its own loan-level data "standard"). Each of them will have to translate their custom templates to the ESMA one and apply rule checks to make sure that the "transformation" worked well (we can provide an example of rule checks that every US bank will have to implement eventually).

The data has to be of high-quality because it is used by the investor for due diligence (before closing) and monitoring (after transaction closing). I simply suggest writing open source connectors and rule checks code: by being open source, the community could double check the mappings (from custom to standard), suggest reviews, as well as add new data quality rule checks (e.g. Interest rate > 0 but < 20).

Loan originators like BMO, Capital One, SocGen, etc. may find this use case interesting.

## Expected features
[TBD]

## Useful apps, systems, projects, technologies, libraries and datasets

- [Morphir](https://morphir.finos.org/)
- ...

**Comment by TheJuanAndOnly99** [comment]
Closing discussion in preparation [for this years hackathon](https://www.finos.org/hosted-events/2024-10-25-citi-india-hackathon-supported-by-finos). Feel free to reopen and update if its still relevant.
